src/main/java/pl/confitura/jelatyna/repository/UserRepository.java

```diff
@@ -0,0 +1,28 @@
+package pl.confitura.jelatyna.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Repository;
+import pl.confitura.jelatyna.domain.User;
+
+import java.util.Optional;
+
+@Repository
+@RepositoryRestResource
+public interface UserRepository extends CrudRepository<User, Long> {
```

Comment:
CrudRepository - zasugerowałem się Participantem który był jako przykład.

save - w drugim branchu dodałem  @NotBlank na adres email i bez spełnienia walidacji nie pozwala dodawać.
kwestia ustalenia co jest wymagane, a co nie (nie uznaliśmy to za kwestię związaną z security)